WATOLINK/bci-gaming
# Boss Transitions

## ClickUp ticket link

[Transition to Boss](https://app.clickup.com/t/8685qf58p)



## Implementation description
The transition is implemented in the scene Health with Heal. Once the boss fight is triggered, there would be a simple
transition from the current scene (Health with Heal) to the boss fight (Boss).
Note: The Transition script includes a possible way to automate all transition in the future. Once all scenes are implemented in the game, then we could modify the build settings to automate transitions between scenes.



## Steps to test
1. Hit play in Health with Heal scene. You should see the screen fade out from red to the game.
2. Then, click your mouse on the screen (since we don't know what would trigger the boss fight yet, this is used as a simple way to test the transition out). You should see the boss fight animation play.
3. When animation finishes, you should be in the Boss scene. Note: the Boss scene would make the character go up and down, this is not part of the transition animation. 



## What should reviewers focus on?
Please make sure that the transition plays successfully and smoothly.
Although the graphics aren't great right now, but later on we could simply change the image to what we want once we have the leisure to make our own graphics.


## Checklist
